sql_metadb/report_queries/erm/count/erm_agreement_count.sql

```diff
@@ -44,7 +45,9 @@ FROM
 
 WHERE 
 	((agrestat.rdv_label = (SELECT agreement_status FROM parameters)) OR 
-		((SELECT agreement_status FROM parameters) = ''))
+		((SELECT agreement_status FROM parameters) = '')
```

Comment:
The one closing parenthesis at the end must be included, since the opening parenthesis is in line 47.